test/moonwall.config.json

```diff
@@ -68,8 +71,9 @@
                             "--reserved-only",
                             "--alice",
                             "--tmp",
-                            "--chain=dancelight-dev",
+                            "--chain=tmp/dancelight-dev-raw.json",
```

Comment:
nice! maybe we should consider it adding it in the zombie tests?

Reply:
Done

*fixed*